docs/UserGuide.md

```diff
@@ -539,50 +472,167 @@ If a parameter is expected only once in the command, but you specified it multip
-
+      <br><br>
 * Add a meeting but there are clashes.
     * `meet 3 -add 20.05.2021 15:30 17:30 MRT`
-
       ![meet-clash](images/meet-clash.png)
 
+
 * Delete a meeting.
     * `meet 1 -delete 20.05.2021 15:00 16:00 MRT`
 
+
 * Clear all meetings.
     * `meet 2 -clear`
 
 [Return to Table of Contents](#table-of-contents)
 <br><br>
 
+### <span style="color:#c93640">Contact Viewing Features</span>
+Contact viewing features allow you to view your client contacts in different ways. For example,
+you may reorder your contact list or look for specific clients.
+
+### <span style="color:#c93640"><code>list</code>: List all clients</span>
+
+**Purpose**: Shows a list of all clients in ClientBook. You may use optional identifiers in conjunction with the minus (-)
+symbol to select which client details you want to see, so that you can have a clutter free view of your client contacts.
+
+**Format**: `list [-p] [-e] [-a] [-i] [-m]`
+
+**Examples**: 
+*  `list` without any specified identifiers shows a list of all clients and all their information.
+
+    ![list](images/list.png)
+<br><br>
+*  One or more identifiers can be used to make `list` only show the specified information. The following command 
+   shows a list of all clients and their phone number and insurance policy number. A client's name and tags will
+   be shown regardless of the identifiers that you specify.
+
+    `list -p -i`
+    
+    ![list phone](images/list-phone-policy-annotate.png)
+
+[Return to Table of Contents](#table-of-contents)
+<br><br>
+
+  
+### <span style="color:#c93640"><code>find</code>: Search for client contact based on keywords</span>
 
-### `addshortcut`: Add shortcut
+**Purpose**: Finds client contacts based on the attribute and keywords that you specify.
+You may use optional identifiers in conjunction with the minus(-) symbol to limit the details shown, similar to the [`list`](#list-list-all-clients) command.
 
-**Purpose**: Adds a shortcut to the existing shortcut library.
+**Format**: `find IDENTIFIER/KEYWORD [& KEYWORDS]…​ [-IDENTIFIER]…​`
+
+* The `IDENTIFIER` specifies which attribute of the client you want to search for. For example, if you want to search for a client contact
+  by name, use the `n` identifier.
+    * For more information about the identifiers for each field, refer to [What information can we store for each client contact?](#what-information-can-we-store-for-each-client-contact).
+* You may not use multiple attributes as your search criteria e.g. you cannot search for a client using address and name at the same time.
+  A command like `find n/David a/Botanic gardens` is invalid.
+* The search is **case-insensitive**. 
+  * E.g. `hans` will match `Hans`.
+* The **order of the keywords does not matter**.
+* The delimiter `&` between keywords allows you to search for Clients using multiple keywords.
+    
+**Examples**:
+* Find `Alex David` and `Alex Yeoh`.
+  * `find n/alex`
+    <br><br>
+* Find `Alex David`.
```

Comment:
Should remove the breaklines after each main bullet since we are not doing so for the other commands.